### PR TITLE
Fe 626 updated cielo logo to go into app

### PR DESCRIPTION
**Pull request by shan8851**
# Description

Update SVG artwork for Cielo logo

---

Fixes # (issue)
https://uniwhales.atlassian.net/browse/FE-626?atlOrigin=eyJpIjoiMzUxMjUxMWIyNzhjNDQ0NDg5ZGEyNDBmODBiMTI5ODciLCJwIjoiaiJ9

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

**Test Configuration**:
* Hardware: Android/iphone/macbook/pc/laptop

## Was this feature tested on all the browsers?

- [X] Chrome/Brave
- [] Firefox
- [] Safari 

# Checklist:

- [X] I have performed a self-review of my code
- [ ] I have tested my changes
- [ ] I have commented my code, particularly in hard-to-understand areas
- [X] My changes generate no new warnings
- [X] Storybook covers all new/pre-existing variations without need to change controls to test it
- [X] I have upgraded version in package.json
- [X] I have assigned QA and Reviewers
- [X] I have labeled the PR accordingly
- [X] I have updated time spent in my jira ticket
- [ ] Build succeeded

---

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@cielo-finance/component-library",
-  "version": "1.0.41",
+  "version": "1.0.42",
```
Comment (goshatravin) [inline package.json:3]: @shan8851 when i do toggle dark/light mode , logo stay without changes 
Screenshot 2022-11-22 at 19 18 55

Comment (shan8851) [reply package.json]: If you switch it in actions you can see it working. 
Screenshot 2022-11-23 at 14 15 53

I wasn't sure how to use theme in storybook, but fixed it so it works now

**Comment by Xtoseone** [comment]
I can't change the logo for dark/white themes

https://user-images.githubusercontent.com/114728833/203575767-7d592afa-d76b-4e3b-bb5b-b1045428c858.mp4

**Comment by shan8851** [comment]
> I can't change the logo for dark/white themes
> 
>  23-11-2022.154446.mp4

So I enabled it to switch via the theme toggle rather than actions. As you can see the text changes colour when you toggle the theme

**Review by ArinCantCode — APPROVED** [review]

**Review by Xtoseone — APPROVED** [review]

**Pull request merged**